src/Tizen.NUI.Components/Utils/StyleManager.cs

```diff
@@ -119,32 +112,30 @@ public void RegisterStyle(string style, string theme, Type styleType, bool bDefa
                 throw new InvalidOperationException($"style can't be null");
             }
 
-            if (theme == null || bDefault == true)
+            if (Activator.CreateInstance(styleType) is StyleBase styleBase)
             {
-                if (defaultStyleSet.ContainsKey(style))
+                var key = "DEFAULT";
+
+                if (theme != null)
                 {
-                    throw new InvalidOperationException($"{style}] already be used");
+                    key = theme.ToUpperInvariant();
+
+                    if (!ThemeMap.ContainsKey(key))
+                    {
+                        ThemeMap[key] = new Theme()
+                        {
+                            Id = theme
+                        };
+                    }
                 }
-                else
+
+                if (ThemeMap[key].HasStyle(style))
                 {
-                    defaultStyleSet.Add(style, Activator.CreateInstance(styleType) as StyleBase);
+                    throw new InvalidOperationException($"{style} already be used");
                 }
-                return;
-            }
 
-            theme = theme.ToUpperInvariant();
-
-            if (themeStyleSet.ContainsKey(style) && themeStyleSet[style].ContainsKey(theme))
-            {
-                throw new InvalidOperationException($"{style}] already be used");
-            }
-
-            if (!themeStyleSet.ContainsKey(style))
-            {
-                themeStyleSet.Add(style, new Dictionary<string, StyleBase>());
+                ThemeMap[key][style] = styleBase.GetViewStyle();
```

Comment:
on FH, sometimes theme of RegisterStyle is null,
so ThemeMap[key] is null sometimes.
if a null check is not added, it causes crash System.NullReferenceException.